projects/plugins/boost/tests/e2e/specs/jetpack-boost/critical-css.test.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import { prerequisitesBuilder } from 'jetpack-e2e-tests/lib/env/prerequisites';
+
+/**
+ * Internal dependencies
+ */
+import JetpackBoostPage from '../../lib/pages/wp-admin/JetpackBoostPage';
+
+// TODO: This is for illustrative purpose only. It will need refactoring and improving.
+describe( 'Critical CSS module', () => {
+	let jetpackBoostPage;
+	beforeAll( async () => {
+		await prerequisitesBuilder().withLoggedIn( true ).withConnection( true ).build();
+	} );
+
+	beforeEach( async function () {
+		jetpackBoostPage = await JetpackBoostPage.visit( page );
+	} );
+
+	afterEach( async function () {
+		if ( jetpackBoostPage ) {
+			await jetpackBoostPage.page.close();
```

Comment:
There's no need to explicitly close the page, it will be closed when tests ends. It's actually important not to close it like this, as the video that's automatically captured won't be saved. A video can only be saved after the page is closed, the whole logic is in playwright-environment.js -> closePage(), linked to the test or hook done events.

Reply:
Got it. Thanks for the explanation 👍 I removed that logic in commit 175c81f3c891880fb89b518409c041666ca583e9